_config.yml

```diff
@@ -1,4 +1,4 @@
 name: Your New Jekyll Site
 markdown: redcarpet
-pygments: true
-baseurl: /figure
+highlighter: null
+baseurl: /figure
```

Comment:
This change will conflict with commit https://github.com/ome/figure/commit/239638c674ea109287a81fbc19ff3e3a624b0593 which is in production only. Is this a requirement?